### DISCUSSION TITLE
Running Pyodide with Wasmtime/Wasmer

### DISCUSSION THREAD

**Discussion by mandesero** [pyodide/pyodide#5145] [Q&A] [answered]
Is it possible to run Pyodide using the Wasmtime runtime or a similar WebAssembly runtime? I don't see a `python*.wasm` file after building Pyodide.

**Comment by ryanking13** [comment, marked as answer]
No. Pyodide is built by the Emscripten toolchain and can only run in a browser or Node.js.

**Reply by hoodmane** [reply]
@mandesero you will likely want to look into the wasi builds of Python. It's a tier 2 supported target and decently well documented.

**Reply by mandesero** [reply]
> @mandesero you will likely want to look into the wasi builds of Python. It's a tier 2 supported target and decently well documented.

I was interested in Pyodide precisely because I need network support, which is not available in, for example, CPython with the WASI SDK or emscripten builds. Thank you, my question is resolved.

**Reply by charles-cai** [reply]
This is also interesting that Pyodide can run in WASM inside a browser but doesn't mean it can run in Wasmtime, a WASM runtime. It would be really nice if Pyodide can run on the server side sandbox like Wasmtime.

**Reply by syrusakbary** [reply]
You can simply use Python compiled to WASIX with Wasmer, should be very easy.

And we support Python fully, check the example on https://wasmer.sh/ @charles-cai
